docs/settings.rst

```diff
@@ -141,6 +141,25 @@ Language Detection
     datetime.datetime(2015, 12, 4, 0, 0)
 
 
+Default Languages
++++++++++++++++++
+
+``DEFAULT_LANGUAGES``: It is a ``list`` of language codes in ISO 639 which is used as default 
+languages and are to be parsed. eg. "en":
+
+    >>> from dateparser import parse
+    >>> parse(settings={'DEFAULT_LANGUAGES': ["en", "es"]})
```

Comment:
I don't think it's right, shouldn't we also pass the string to be parsed?

Reply:
I also find this example weird, it should have a `date_string`

Reply:
Thanks, I have made the changes 👍🏻